orc8r/cloud/go/services/tenants/obsidian/models/swagger.v1.yml

```diff
@@ -114,3 +146,17 @@ definitions:
         type: array
         items:
           type: string
+  control_proxy:
+    type: object
+    required:
+      - id
+      - control_proxy
+    properties:
+      id:
```

Comment:
Unclear that we need ID in this object -- since we're only PUT-ing, we can grab the ID from the PUT endpoint

Reply:
We could, but I wanted to be same as `tenant` (see L130-148) 

Thoughts?

Reply:
I generally prefer conciseness (e.g. don't repeat a value in both the key and value), but I also get that there's benefit to a value being able to self-identify. I'll let you make the decision then

Reply:
Maybe I can make an issue for removing `id` for both?